Examples/Scripts/Python/seeding.py

```diff
@@ -103,49 +103,86 @@ def runSeeding(trackingGeometry, field, outputDir, s=None):
         ),
     )
 
-    gridConfig = acts.SpacePointGridConfig(
```

Comment:
create new separate orthogonalSeeding.py and keep seeding.py?

Reply:
No strong opinions here.

Reply:
Yeah, the executable now takes --orthogonal flag which uses the new strategy, if the flag is not provided it will use the old strategy. :slightly_smiling_face: